Library/Formula/saldl.rb

```diff
@@ -0,0 +1,43 @@
+class Saldl < Formula
+  desc "CLI downloader optimized for speed and early preview."
+  homepage "https://saldl.github.io"
+  url "https://github.com/saldl/saldl/archive/v31.tar.gz"
+  sha256 "146a980ae2109a391c7a8ab6a1c525458db9bf18f3f0477731c9b876630078b4"
+
+  head do
+    # git describe does not work with shallow clones
+    url "https://github.com/saldl/saldl.git", :shallow => false
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "asciidoc" => :build
+  depends_on "libevent"
+
+  # curl >= 7.42 is required
+  depends_on "curl" if MacOS.version <= :mavericks
+
+  def install
+    # a2x/asciidoc needs this to build the man page successfully
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+
+    args = [
```

Comment:
Just do `args = ["--prefix=#{prefix}"]` rather than having a block for one line.